.github/BOTMETA.yml

```diff
@@ -1024,7 +1024,7 @@ files:
   $modules/system/alternatives.py:
     maintainers: mulby
```

Comment:
Would be glad if you would consider to add yourself here instead of at the ignore list :) Then you'd be one of the [module maintainers](https://docs.ansible.com/ansible/devel/community/maintainers_workflow.html#module-maintainers) (resp. probably the only active one since right now nobody seems to be active) for the alternatives module, and would be informed of issues and new PRs for that module.